includes/controller/guest_login_controller.php

```diff
@@ -314,12 +314,12 @@ function guest_register() {
                   )),
                   div('row', array(
                   div('col-sm-8', array(
-                    form_select('native_lang', _("Native Language").' '. entry_required(), languages(), $native_lang, 'English')
```

Comment:
The variable is not defined while showing the view, hence error is produced.
The form_select method has only 5 parameters, 6 are used